recipes/pudb/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2016.2" %}
+
+package:
+  name: pudb
+  version: {{ version }}
+
+source:
+  fn: pudb-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pudb/pudb-{{ version }}.tar.gz
+  md5: 4573b70163329c1cb59836a357bfdf7c
+
+build:
+  # conda skeleton added this, but I'm unsure why it would be needed.
+  # preserve_egg_dir: True
```

Comment:
That is strange. I forget exactly the conditions that this gets triggered under. I do remember investigating it before though.